### PR TITLE
RC: Maintenance windows | update docs

### PR DESCRIPTION
**Pull request by cmilesb**
DOC-3496
Staging link: https://docs.redis.com/staging/DOC-3496/rc/subscriptions/maintenance/

### CODE REVIEW

File: content/rc/subscriptions/maintenance/_index.md [hunk 1]
```diff
@@ -26,15 +26,32 @@ This includes, but is not limited to:
 - Adding more memory to a node
 - Applying security patches
 
-Redis will notify users by email when maintenance starts and ends. If Redis needs an action from a user to start maintenance, Redis will notify users with a reasonable amount of time before planned maintenance.
-
-For major upgrades or upgrades that might include breaking changes, users will receive an advance notification with sufficient time to prepare before the upgrade.
+Redis will notify users by email when maintenance starts and ends. For more details, see [Notifications](#notifications).
 
-If you want to receive notifications by email, make sure **Operational emails** are activated in your user settings in [Access Management]({{<relref "/rc/security/access-control/access-management">}}). 
+During maintenance, your application may disconnect from our service for a few seconds. Most Redis clients are set to refresh their DNS address when they reconnect to the database, and you will not be required to perform any further action. If you encounter connectivity problems for more than a minute during maintenance, please refresh your DNS entries.
```
Comment (NoamSternRedis) [inline content/rc/subscriptions/maintenance/_index.md:31]: Besides my previous comment about adding the point about the latency, I suggest starting the section with "during a maintenance activity, your database will be operational, but...

**Review by NoamSternRedis — CHANGES_REQUESTED** [review]

File: content/rc/subscriptions/maintenance/_index.md (hunk 1 above)
Comment (cmilesb) [reply content/rc/subscriptions/maintenance/_index.md]: Fixed in [1bbee48](https://github.com/RedisLabs/redislabs-docs/pull/3194/commits/1bbee482e5a922619d09fe502aaa46a4107c6170).

File: content/rc/subscriptions/maintenance/_index.md [hunk 2]
```diff
@@ -12,7 +12,7 @@ aliases:
 
 Redis will maintain your Redis Cloud subscriptions and databases as needed to ensure your databases are running the most stable and up-to-date version of Redis. During maintenance, you may notice some latency when connecting to your databases.
```
[the remark this answers is not in the document]
Comment (cmilesb) [reply content/rc/subscriptions/maintenance/_index.md]: Fixed in [1bbee48](https://github.com/RedisLabs/redislabs-docs/pull/3194/commits/1bbee482e5a922619d09fe502aaa46a4107c6170).

File: content/rc/subscriptions/maintenance/_index.md [hunk 3]
```diff
@@ -26,15 +26,32 @@ This includes, but is not limited to:
...
 
-Redis will notify users by email when maintenance starts and ends. If Redis needs an action from a user to start maintenance, Redis will notify users with a reasonable amount of time before planned maintenance.
-
-For major upgrades or upgrades that might include breaking changes, users will receive an advance notification with sufficient time to prepare before the upgrade.
+Redis will notify users by email when maintenance starts and ends. For more details, see [Notifications](#notifications).
 
-If you want to receive notifications by email, make sure **Operational emails** are activated in your user settings in [Access Management]({{<relref "/rc/security/access-control/access-management">}}). 
+During maintenance, your database will be operational, but you may notice some latency when connecting to your databases. Your application may also disconnect from your database for a few seconds. Most Redis clients are set to refresh their DNS address when they reconnect to the database, and you will not be required to perform any further action. If you encounter connectivity problems for more than a minute during maintenance, please refresh your DNS entries.
```
Comment (NoamSternRedis) [inline content/rc/subscriptions/maintenance/_index.md:31]: @cmilesb, maybe we should bold it in some way and not just embed it in the paragraph

File: content/rc/subscriptions/maintenance/_index.md [hunk 4]
```diff
@@ -26,15 +26,32 @@ This includes, but is not limited to:
...
 
 ### Urgent maintenance
 
 Urgent maintenance refers to any activity that could affect service and cannot wait for scheduling. This includes applying urgent security patches.
 
 Redis can perform urgent maintenance at any time, even if you have set a manual maintenance window or have temporarily skipped maintenance. Redis will notify users by email when urgent maintenance starts and ends.
```
Comment (NoamSternRedis) [inline content/rc/subscriptions/maintenance/_index.md:37]: hyperlink on the words:  "temporarily skipped maintenance" to the relevant section - https://docs.redis.com/staging/DOC-3496/rc/subscriptions/maintenance/set-maintenance-windows/#skip-maintenance-temporarily

**Review by NoamSternRedis — CHANGES_REQUESTED** [review]

File: content/rc/subscriptions/maintenance/_index.md (hunk 3 above)
Comment (cmilesb) [reply content/rc/subscriptions/maintenance/_index.md:31]: @NoamSternRedis Bolding or italicizing entire sections of text doesn't fit our style. I separated the "disconnection" part from the "latency" part into two separate paragraphs. Do you think that's enough, or should I use a note to bring attention to the disconnection? Or, is there a specific part of this section you think needs to be bolded?

File: content/rc/subscriptions/maintenance/_index.md [hunk 5]
```diff
@@ -26,15 +26,34 @@ This includes, but is not limited to:
...
 ### Urgent maintenance
 
 Urgent maintenance refers to any activity that could affect service and cannot wait for scheduling. This includes applying urgent security patches.
 
-Redis can perform urgent maintenance at any time, even if you have set a manual maintenance window or have temporarily skipped maintenance. Redis will notify users by email when urgent maintenance starts and ends.
+Redis can perform urgent maintenance at any time, even if you have set a manual maintenance window or have temporarily [skipped maintenance]({{<relref "/rc/subscriptions/maintenance/set-maintenance-windows#skip-maintenance-temporarily">}}). Redis will notify users by email when urgent maintenance starts and ends.
+
+## Notifications
+
+Redis will notify users by email when maintenance starts and ends. If Redis needs an action from a user to start maintenance, Redis will notify users with a reasonable amount of time before planned maintenance.
```
Comment (mich-elle-luna) [inline content/rc/subscriptions/maintenance/_index.md:43]:
```suggestion
Redis will notify you by email when maintenance starts and ends. If Redis needs an action from you to start maintenance, Redis will notify you with a reasonable amount of time before planned maintenance.
```

File: content/rc/subscriptions/maintenance/_index.md [hunk 6]
```diff
@@ -26,15 +26,34 @@ This includes, but is not limited to:
...
+
+For major upgrades or upgrades that might include breaking changes, users will receive an advance notification with sufficient time to prepare before the upgrade.
```
Comment (mich-elle-luna) [inline content/rc/subscriptions/maintenance/_index.md:45]:
```suggestion
For major upgrades or upgrades that might include breaking changes, you will receive an advance notification with sufficient time to prepare before the upgrade.
```

File: content/rc/subscriptions/maintenance/_index.md [hunk 7]
```diff
@@ -26,15 +26,34 @@ This includes, but is not limited to:
...
+Your application may also disconnect from your database for a few seconds. Most Redis clients are set to refresh their DNS address when they reconnect to the database, and you will not be required to perform any further action. If you encounter connectivity problems for more than a minute during maintenance, please refresh your DNS entries.
 
 ### Urgent maintenance
 
 Urgent maintenance refers to any activity that could affect service and cannot wait for scheduling. This includes applying urgent security patches.
 
-Redis can perform urgent maintenance at any time, even if you have set a manual maintenance window or have temporarily skipped maintenance. Redis will notify users by email when urgent maintenance starts and ends.
+Redis can perform urgent maintenance at any time, even if you have set a manual maintenance window or have temporarily [skipped maintenance]({{<relref "/rc/subscriptions/maintenance/set-maintenance-windows#skip-maintenance-temporarily">}}). Redis will notify users by email when urgent maintenance starts and ends.
```
Comment (mich-elle-luna) [inline content/rc/subscriptions/maintenance/_index.md:39]:
```suggestion
Redis can perform urgent maintenance at any time, even if you have set a manual maintenance window or have temporarily [skipped maintenance]({{<relref "/rc/subscriptions/maintenance/set-maintenance-windows#skip-maintenance-temporarily">}}). Redis will notify you by email when urgent maintenance starts and ends.
```

**Review by mich-elle-luna — APPROVED** [review]
Thank you, just suggesting a change from users --> you for style.

**Pull request merged**